### ISSUE TITLE
Getting Klebsiella all the time

### ISSUE THREAD

**Issue by alanxelena** [DerrickWood/kraken2#763]
Dear everyone,

  I've been running Kraken on a couple of samples, as I noted in a question before, I was having some curious results so I run Kraken2/Bracken on a mock community (SRR8073716) using the silva Database. The classification was 90% spot on with only one missing genus but I noticed a fair amount of Klebsiella. This was also the case in my previous analyses but it didn't quite pop out because I was analysing human related samples and it made sense. I saw this behaviour using different confidence values (0.1, 0.3 and 0.5)
My question here is, is there a known false positive call for Klebsiella? I couldn't find anything related to the database. 

Thanks!

**Comment by jenniferlu717** [comment]
I don't know of any known false positive for Klebsiella but have you tried blasting any of the reads that were classified as Klebsiella? 

You can try aligning to a Klebsiella genome or extract the reads (with KrakenTools https://github.com/jenniferlu717/KrakenTools) and blast them to see if they still hit Klebsiella.

**Comment by alanxelena** [comment]
Hi Jennifer,

  This was a while ago but I only had time to analyse this today. I did what you suggested and got 1568 sequences (only for the forward read) that were classified as Klebsiella, I did some random blasting of a couple of them and in every case I get only one Klebsiella hit and 99 Escherichia coli. The Klebsiella is not even the top hit. I wonder why I'm getting this weird classification
